src/main/java/com/easypost/model/beta/ReferralCustomer.java

```diff
@@ -19,6 +19,10 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * @deprecated Use {@link com.easypost.model.ReferralCustomer} instead.
+ * Deprecated: v5.10.0 - v7.0.0
```

Comment:
Issue: for beta features, we won't use the traditional "2 version removal" policy, since we can't guarantee the beta feature itself at the API level will live on that long. Once the API beta endpoint is gone, we are free to remove from the lib since it will become unusable. For that reason, let's not tag these with the version ranges.

If this was a deprecated non-beta feature, than the tags would absolutely apply.

Reply:
Resolved